addons/sale_timesheet/models/project.py

```diff
@@ -174,6 +175,16 @@ def _compute_sale_line_id(self):
             ], limit=1)
             project.sale_line_id = sol or project.sale_line_employee_ids.sale_line_id[:1]  # get the first SOL containing in the employee mappings if no sol found in the search
 
+    @api.depends('analytic_account_id')
+    def _compute_sales_orders_count(self):
+        sales_orders_data = self.env['sale.order'].read_group([
+                ('analytic_account_id', '!=', False),
+                ('analytic_account_id', 'in', self.analytic_account_id.ids)
+        ], ['analytic_account_id'], ['analytic_account_id'])
+        mapped_data = {data['analytic_account_id'][0]: data['analytic_account_id_count'] for data in sales_orders_data}
+        for project in self:
+            project.sales_orders_count = mapped_data.get(project.analytic_account_id.id, 0)
```

Comment:
Great ! 🤩 
Small question: Are we sure an AA is linked to only a project and not many projects ? 🤔